.config/tsaoptions.json

```
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\mono",
```

I think we should use this area path instead:

```suggestion
    "areaPath": "DevDiv\\NET Runtime\\Managed Linker",
```